content/Tjenester/fiksprotokoll/_index.md

```diff
@@ -168,41 +162,20 @@ Tilsvarende for å fjerne forespørsel
 
 `DELETE {fiksOrgId}/systemer/{systemId}/forespurteTilganger/{eksternKonto}`
 
+[OpenAPI Specification (Fiks Protokoll API)](https://editor.swagger.io/?url=https://developers.fiks.ks.no/api/fiks-protokoll-konfigurasjon-api-v1.json)
 
 
 Merk: Søk etter systemer er ikke implementert ennå, så det kan være vanskelig å finne systemer å be om tilgang til
 
 #### Kontoer et system kan sende meldinger til
-Hvilke kontoer et system kan sende meldinger til finnes på system-responsen fra `GET /fiks-protokoll/api/v1/konfigurasjon/{fiksOrgId}/systemer/{systemId}`. Der vil `tilgangTilKontoer` ha en liste over kontoer (`EksternProtokollKontoResponseEksternProtokollKontoResponse`) systemet kan sende meldinger til.
+API'et støtter å hente hvilke kontoer et system kan sende meldinger til.
 Kun kontoer som er parter fra samme protokoll (samme navn og versjon) og som er støttet part (er i listen av stottedeParter hos motaker) som kan sende meldinger til hverandre. Dersom et system ikke ha en konto som er støttet part, så må en slik konto opprettes før meldinger kan sendes. 
 
-Definisjon av `EksternProtokollKontoResponseEksternProtokollKontoResponse`:
-```json
-{
-  "id": "ID på kontoen. Det er denne IDen Fiks IO skal sende til",
-  "navn": "Navn på konto",
```

Comment:
Jeg ville inkludert et JSON eksempel her. Ser den at det er sårbart for endringer om vi endrer specen, men problemet i utgangspunktet var jo at kunden ikke fulgte lenken til api spec, men så på eksempel JSON.

Reply:
For dokumentasjons purposes er det jo også lettere kanskje å ikke åpne en swagger spec og myse seg frem til endepunktet man skal kode mot. Bare se på en eksempel JSON og boom, okei, der har vi'n

Reply:
Joda, jeg ser poenget ditt, men de burde gå inn på Swagger for det er der sannheten vil være. Og man ser mer info? Jeg tenker at vi ellers peker til Swagger i dokumentasjonen vår også. 
Med Swagger kan de teste det mer korrekt. Han burde gått inn på Swagger og nå tillater vi ikke noen å være "lat".

Reply:
Fair enough. Ikke uenig på generelt grunnlag :)